### PR TITLE
add test and change base image

### PR DESCRIPTION
**Pull request by bdwyer2**
- Switched base image to `codait/max-base`
- Added tests, due to memory limits with Travis CI these test can only be run locally

### CODE REVIEW

File: tests/test.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    c.perform()
+    assert c.getinfo(pycurl.RESPONSE_CODE) == 200
+    c.close()
+
+    response = b.getvalue()
+    response = json.loads(response)
+
+    assert response['status'] == 'ok'
+
+    print("output: " + response['pred_txt'][0]['pred_txt'])
```
Comment (bdwyer2) [inline tests/test.py]: Currently there is no way to set a random seed so the output text will be different for each run.

**Comment by bdwyer2** [comment]
This is passing locally. I think it's running out of memory or something on Travis. Will investigate more.

**Comment by bdwyer2** [comment]
Looks like Travis CI has only provides 7.5 GB of RAM and this model requires about 12 GB.

**Comment by bdwyer2** [comment]
@MLnick I removed Travis CI since I don't see any work around for the memory limits. At least we will be able to test the model locally and possibly on our on CI server.

**Comment by MLnick** [comment]
Ok for now.

I think the only real solution will be to try limit the model size - if I
recall it is around 800,000 word vectors? We could limit it to the most
common X words to try limit the number of model parameters

**Comment by bdwyer2** [comment]
@MLnick okay. I will merge this in and then look into doing that.

**Pull request merged**